cli_tools_e2e_test/gce_image_import_export/test_suites/import/import_tests.go

```diff
@@ -96,14 +96,36 @@ var cases = []testCase{
 		source:        "projects/compute-image-tools-test/global/images/debian-9-translate",
 		os:            "opensuse-15",
 		expectedError: "\"debian-9\" was detected on your disk, but \"opensuse-15\" was specified",
+	},
+	// EL
+	{
+		caseName: "el-centos-7-8",
+		source:   "projects/compute-image-tools-test/global/images/centos-7-8",
+		os:       "centos-7",
+	}, {
+		caseName: "el-centos-8-0",
+		source:   "projects/compute-image-tools-test/global/images/centos-8-import",
+		os:       "centos-8",
+	}, {
+		caseName: "el-centos-8-2",
+		source:   "projects/compute-image-tools-test/global/images/centos-8-2",
+		os:       "centos-8",
 	}, {
-		caseName:  "rhel-7-uefi",
+		caseName:  "el-rhel-7-uefi",
```

Comment:
Since`el` is already included in `rhel`, why the duplication in case names? Is it just for sorting purposes?